usdoj-crt/beta-ada
# Role="status" not working in some areas - JS Fix [WIP]

Role status is now working on the Guidance and Resources page. It will not work consistently on the search page as that page does a full refresh on search. 

### Testing:
1. Head to the Guidance and Resources page
2. Turn on MacOS voiceover - https://support.apple.com/guide/voiceover/turn-voiceover-on-or-off-vo2682/mac
3. Toggle the checkboxes
4. The Screen reader should announce the number of items returned.
